english/marshmallow.md

```diff
@@ -10,7 +10,13 @@ usually take intoxicants, what's happening...?
 
 But it's probably not worth dwelling on that right now, you could be in danger.
 
-Grave danger.
+Grave danger.  
+
+The type of mortal danger that so often accompanies confectionary inspired interior design.
+
+Quickly surveying your immediate surrounds, you notice a garishly dressed, vacant eyed young man lounging on a nearby sofa.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.